src/pages/NewsAdd.jsx

```diff
@@ -0,0 +1,152 @@
+import { useState } from "react"
+import dayjs from "dayjs"
+import "dayjs/locale/ko"
+dayjs.locale("ko")
+
+import { Stack, styled } from "@mui/system"
+import AddIcon from "@mui/icons-material/Add"
+import ArrowLeftIcon from "@mui/icons-material/ArrowLeft"
+import ArrowRightIcon from "@mui/icons-material/ArrowRight"
+import Chip from "@mui/material/Chip"
+import TextField from "@mui/material/TextField"
+
+import Layout from "../components/Layout/Layout"
+import { IconButton, Typography } from "@mui/material"
+
+import { collection, addDoc } from "firebase/firestore"
+import { db } from "../firebase" // firebase 설정 파일을 임포트
+
+const NewsAdd = () => {
+  const today = dayjs()
+  const dayOfWeek = today.day() // 0은 일요일, 6은 토요일
+  const lastSunday = today.subtract(dayOfWeek, "day")
+  const [selectedDateInfo, setSelectedDateInfo] = useState(lastSunday)
+  const [textValue, setTextValue] = useState("") // 텍스트 상태 추가
+
+  const handlePreviousMonth = () => {
+    const newDateInfo = selectedDateInfo.subtract(7, "day")
+    setSelectedDateInfo(newDateInfo)
+  }
+
+  const handleNextMonth = () => {
+    const newDateInfo = selectedDateInfo.add(7, "day")
+    setSelectedDateInfo(newDateInfo)
+  }
```

Comment:
```suggestion
  const handlePreviousMonth = () => {
    const newDateInfo = selectedDateInfo.subtract(7, "day")
    setSelectedDateInfo(newDateInfo)
  }

  const handleNextMonth = () => {
    const newDateInfo = selectedDateInfo.add(7, "day")
    setSelectedDateInfo(newDateInfo)
  }
```
특정 일자의 추가/수정에서는 다른 달로 이동하면 안될 것 같은데? 
내용 추가/수정 부분과 조회 부분이 분리가 되어야 할 것 같음

Reply:
추가 페이지 
1) 일자 변경을 통해서 원하는 일자의 소식을 업데이트 하기 위함입니다!
수정 페이지
1) 일자 변경을 통해서 해당 일자의 소식을 텍스트 필드에 가져와서 빠르게 수정하기 위함입니다!

Reply:
@dony0720 그렇게할거면 수정 버튼을 추가 버튼 옆에 넣거나 해야할듯
지금 UI 상에서는 해당 일자의 광고마다 수정/삭제 버튼이 있기때문에 수정버튼을 누르는 액션은 해당 날짜의 내용을 수정하는것으로 고려하면 될듯